src/webgpu/api/validation/encoding/encoder_open_state.spec.ts

```diff
@@ -260,8 +262,10 @@ g.test('non_pass_commands')
           }
           break;
         case 'writeTimestamp':
-          {
+          try {
             encoder.writeTimestamp(querySet, 0);
+          } catch (ex) {
+            t.skip('writeTimestamp is actually not available');
```

Comment:
Should this be `t.skipIf(ex instanceof TypeError, ...)`? If so, update other `t.skip('writeTimestamp is actually not available');` in other files.

Reply:
Good idea, will do.